### PR TITLE
Spell check the documentation

### PR DESCRIPTION
**Pull request by marchdf**
## Summary

Spellcheck the documentation.

## Pull request type

Please check the type of change introduced:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: docs/sphinx/spelling-wordlist.txt [hunk 1]
```diff
@@ -0,0 +1,127 @@
+Burggraf
+CFL
+CMake
+CTest
+Ekman
+Euler
+Exascale
+Exascale
```
Comment (moprak-nrel) [inline docs/sphinx/spelling-wordlist.txt:8]: Repeated Exascale

File: docs/sphinx/spelling-wordlist.txt [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+Nalu
+Navier
+Navier
+Neumann
+Nicolson
+Obukhov
+Poiseuille
+Raphson
+Shubin
+Viscosities
```
Comment (moprak-nrel) [inline docs/sphinx/spelling-wordlist.txt:23]:
```suggestion
viscosities
```

File: docs/sphinx/spelling-wordlist.txt [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+dt
+enstrophy
+exawind
+executables
+fidelities
+fllc
+freestream
+frontend
+geostrophic
+godunov
```
Comment (moprak-nrel) [inline docs/sphinx/spelling-wordlist.txt:67]:
```suggestion
Godunov
```

File: docs/sphinx/spelling-wordlist.txt [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+hexahedral
+hypre
+incflo
+incompressibility
+initializer
+initializers
+iteratively
+kosovic
```
Comment (moprak-nrel) [inline docs/sphinx/spelling-wordlist.txt:75]:
```suggestion
Kosovic
```

File: docs/sphinx/spelling-wordlist.txt [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+overset
+paraview
+perturbational
+petsc
+piecewise
+plotfile
+plotfiles
+plt
+postprocessing
+prandtl
```
Comment (moprak-nrel) [inline docs/sphinx/spelling-wordlist.txt:97]:
```suggestion
Prandtl
```

File: docs/sphinx/spelling-wordlist.txt [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+pre
+priori
+proj
+reStructuredText
+regrid
+regridding
+repo
+roadmap
+rosco
+smagorinsky
```
Comment (moprak-nrel) [inline docs/sphinx/spelling-wordlist.txt:107]:
```suggestion
Smagorinsky
```

File: docs/sphinx/spelling-wordlist.txt [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+Navier
+Neumann
+Nicolson
+Obukhov
+Poiseuille
+Raphson
+Shubin
+Viscosities
+Zalesak
+abl
```
Comment (moprak-nrel) [inline docs/sphinx/spelling-wordlist.txt:25]: ABL?

File: docs/sphinx/spelling-wordlist.txt [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+Exascale
+Exascale
+Fortran
+Joukowsky
+MPI
+Moeng
+Monin
+Nalu
+Navier
+Navier
```
Comment (moprak-nrel) [inline docs/sphinx/spelling-wordlist.txt:16]: repeated Navier

**Review by moprak-nrel — COMMENTED** [review]
Looks great! Have a few minor changes in the spell list.

**Review by moprak-nrel — APPROVED** [review]

File: docs/sphinx/spelling-wordlist.txt [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+concretize
+concretized
+concretizing
+conda
+convecting
+cpp
+datafile
+dimensionalize
+discretization
+discretization
```
Comment (moprak-nrel) [inline docs/sphinx/spelling-wordlist.txt:50]: dupe

File: docs/sphinx/spelling-wordlist.txt [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+discretized
+discretized
```
Comment (moprak-nrel) [inline docs/sphinx/spelling-wordlist.txt:52]: dupe

File: docs/sphinx/spelling-wordlist.txt [hunk 11]
```diff
@@ -0,0 +1,125 @@
...
+solvability
+spack
+spanwise
+spinup
+src
+subgrid
+submodule
+submodules
+teardown
+th
```
Comment (moprak-nrel) [inline docs/sphinx/spelling-wordlist.txt:115]: does `th` refer to something universal?

File: docs/sphinx/spelling-wordlist.txt [hunk 12]
```diff
@@ -0,0 +1,125 @@
...
+iteratively
+Kosovic
+levelset
+lidar
+mesoscale
+microscale
+multigrid
+namespace
+namespaces
+nd
```
Comment (moprak-nrel) [inline docs/sphinx/spelling-wordlist.txt:81]: Curious if `nd` should be here?

**Review by moprak-nrel — COMMENTED** [review]
Couple more minor things in the spell list

File: docs/sphinx/spelling-wordlist.txt (hunk 12 above)
Comment (marchdf) [reply docs/sphinx/spelling-wordlist.txt:81]: same comment as above

File: docs/sphinx/spelling-wordlist.txt (hunk 11 above)
Comment (marchdf) [reply docs/sphinx/spelling-wordlist.txt:115]: `nd` and `th` are used in stuff like `2nd` and `4th`. But maybe it's better for me to track those down and just change that usage. Let me see.

**Comment by marchdf** [comment]
eagle eyes, @moprak-nrel, eagle eyes. Thank you

File: docs/sphinx/spelling-wordlist.txt (hunk 11 above)
Comment (marchdf) [reply docs/sphinx/spelling-wordlist.txt]: there were so few, I just turned them into actual words.

**Pull request merged**